envisage/plugins/ipython_kernel/internal_ipkernel.py

```diff
@@ -3,11 +3,18 @@
 https://github.com/ipython/ipython/blob/2.x/examples/Embedding/internal_ipkernel.py
 
 """
+from distutils.version import StrictVersion as Version
+
+import ipykernel
 from ipykernel.connect import connect_qtconsole
 from ipykernel.kernelapp import IPKernelApp
 
 from traits.api import Any, HasStrictTraits, Instance, List
 
+from tornado import ioloop
```

Comment:
Nitpick: can we move this import into the 3rd party imports block (i.e., just below the `ipykernel` imports, with no intervening blank line)?

Reply:
done